core/file/atime_spec.rb

```diff
@@ -15,6 +15,17 @@
     File.atime(@file).should be_kind_of(Time)
   end
 
+  platform_is :linux do
+    it "returns the last access time for the named file with microseconds" do
+      3.times do
```

Comment:
I think it's OK as it is (that's why I merged it), but if you want to change please feel free to open another Pull Request.